n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -283,6 +286,7 @@ def run(self):
                 replicas = [replica]
             self.environment.replicas = NSList(replicas, nskey="replica")
             self.environment.hyperopt = self.args["hyperopt"]
+            self.environment.restart = self.args["continue"]
```

Comment:
Is there a reason you go from "continue" to "restart"? Using different names for the same thing can make code more difficult to read.

Reply:
Yes, I agree that it makes more sense indeed to just use `--restart` instead of `--continue`. If this is OK, I will make the changes.

Reply:
Sure, please go ahead.